src/Workspaces/Core/Portable/Storage/SQLite/Interop/SqlConnection.cs

```diff
@@ -69,13 +69,17 @@ public static SqlConnection Create(IPersistentStorageFaultInjector faultInjector
 
             raw.sqlite3_busy_timeout(handle, (int)TimeSpan.FromMinutes(1).TotalMilliseconds);
 
-            return new SqlConnection(faultInjector, handle);
+            var queryToStatement = new Dictionary<string, SqlStatement>();
```

Comment:
i wouldn't mind a comment here as well.

Reply:
I already have a comment in one location, and it would be impossible to change this location without seeing the other location. I prefer to not duplicate.